Tests/ValetIntegrationTests/SecureEnclaveIntegrationTests.swift

```diff
@@ -199,4 +202,106 @@ class SecureEnclaveIntegrationTests: XCTestCase
             XCTAssertEqual(try otherValet.string(forKey: key), value)
         }
     }
+
+    // MARK: string(forKey:withPrompt:)
+
+    func test_stringForKeyWithPrompt_throwsItemNotFoundForKeyWithNoValue() throws {
+        XCTAssertThrowsError(try valet.string(forKey: key, withPrompt: prompt)) { error in
+            XCTAssertEqual(error as? KeychainError, .itemNotFound)
+        }
+    }
+
+    func test_stringForKeyWithPrompt_retrievesStringForValidKey() throws {
+        try valet.setString(passcode, forKey: key)
+        XCTAssertEqual(passcode, try valet.string(forKey: key, withPrompt: prompt))
+    }
+
+    // MARK: string(forKey:withPrompt:withFallbackTitle:)
+
+    func test_stringForKeyWithPromptWithFallbackTitle_usesCorrectPolicy() throws {
+        // SecureEnclaveAccessControl.userPresence should map to LAPolicy.deviceOwnerAuthentication
+        do {
+            let valet = SecureEnclaveValet.valet(with: type(of: self).identifier, accessControl: .userPresence)
+            try valet.setString(passcode, forKey: key)
+            let authContext = MockLAContext(reply: (true, nil))
+            valet.authenticationContextProvider = { authContext }
+            _ = try valet.string(forKey: key, withPrompt: prompt, withFallbackTitle: fallback)
+
+            XCTAssertEqual(authContext.evaluatePolicyCalls.count, 1)
+            XCTAssertEqual(authContext.evaluatePolicyCalls.first, .deviceOwnerAuthentication)
+        }
+
+        // SecureEnclaveAccessControl.biometricCurrentSet should map to LAPolicy.deviceOwnerAuthenticationWithBiometrics
```

Comment:
Looks like we're running two separate test runs. Let's split this test into two? I think that'd help name the test more explicitly.

(If you're following prior art elsewhere in this file, my apologies! My thoughts on how tightly to scope test methods has changed over time. Shout-out to @fdiaz for pushing me towards smaller tests)